### ISSUE TITLE
Test reporting

### ISSUE THREAD

**Issue by schanzer** [ds26gte/code.pyret.org#93]
When all tests pass, CPO reads the message aloud -- BEAUTIFUL!
But if any test fails, the editor reads `Loading definitions window produced output. 1`

**Comment by ds26gte** [comment]
Fixed in 7e9fa12

**Comment by schanzer** [comment]
Since it looks like you're grabbing the innerText of the whole test result DOM, I understand that your options are limited. Is there no way to get the raw error report struct?

If you could, I'd suggest adding periods and commas to make things read more smoothly: "1 test passed - _comma_ - one test failed - _period_ - examples-block-1 - _colon_ - 1 out of 2 tests passed in this block".

Or, even better, start adding/dropping words: "_in_ examples-block-1-_color_- 1 out of 2 tests passed".

But if this is not easy, we can call it ready for 1.0.

**Comment by ds26gte** [comment]
Added improvements in 609b251

**Comment by schanzer** [comment]
Beautiful. This is looking really slick!
